### PR TITLE
Sclang/print version

### PR DESCRIPTION
**Pull request by blacksound**
Related to #1322
Adding same functionality for sclang for consistency.

### CODE REVIEW

File: lang/LangSource/SC_TerminalClient.cpp [hunk 1]
```diff
@@ -166,6 +168,11 @@ bool SC_TerminalClient::parseOptions(int& argc, char**& argv, Options& opt)
 			case 'r':
 				opt.mCallRun = true;
 				break;
+			case 'V':
+				fprintf(stdout, "sclang %s\n", SC_VersionString().c_str());
+				quit(0);
+				return false;
+				break;
```
Comment (danstowell) [inline lang/LangSource/SC_TerminalClient.cpp]: "return false" and then "break"? this looks odd. break not needed, at least, since unreachable

**Comment by blacksound** [comment]
@danstowell 
How about this change? 86f88b275dd47b748f9bc14a381fb0dbd0647b11
It is more in the same vein as the rest of the code at least.

**Comment by gusano** [comment]
Same comment here as https://github.com/supercollider/supercollider/pull/1322#issuecomment-75513005

**Comment by danstowell** [comment]
@blacksound no, don't add a goto for no reason, just keep it as it is

**Comment by blacksound** [comment]
@danstowell
The reason I added it was that other cases in the switch statement uses them, so it would be more consistent with the style in general.
But if you think that I should revert back to the previous commit, I can do that. Just let me know.

**Comment by danstowell** [comment]
@blacksound yes, no problem, I understand why you did it - but really we do not want to add a goto that is used only in one place - yes, do revert

**Comment by blacksound** [comment]
@danstowell:  reverted now. Hope it's fine now. Built and tested ok on my end.

Maybe more should weigh in on the -V/-v thing that @gusano raised?

**Comment by crucialfelix** [comment]
-v is much more common

--version is the normal alternate

-V suggests verbose

I detest having to run help or google just to figure out how to find out what version a binary is.

**Comment by gusano** [comment]
> -v is much more common
> --version is the normal alternate
> -V suggests verbose

:+1: 

Just a thought, could it be an option to change `-V` for *verbose* and use `-v | --version` for version? (I have no idea if *verbose* is commonly used).

**Comment by blacksound** [comment]
Changing -v to -V probably won't cause an insurmountable amount of code breakage for decades to come neither.

It would be nice if both sclang and scsynth have the same option for the same thing.

The option parser is implemented for single dash options for the moment AFAIK.
A more experienced dev can most likely expand my humble contribution to implement parsing double dash options, with much greater ease than me.

**Comment by timblechmann** [comment]
boost program_options might ease things:
http://www.boost.org/doc/libs/1_57_0/doc/html/program_options/overview.html

however i'd strongly suggest not to change the semantics of command line arguments!

**Comment by danstowell** [comment]
In general I agree that backward compatibility is better, but I doubt that the current -v is widely used. I implemented it and used it for some of my projects, personally I don't object to the switchround. I don't think there's a particularly forceful argument either way, so someone should make a decision and go with it :)

**Comment by gusano** [comment]
I vote for switching:
- `v` for version
- `V` for verbose (which is probably muss less used)

**Comment by gusano** [comment]
If you agree but feel lazy, here's a patch:
https://github.com/gusano/supercollider/commit/230a00c58161b8aa956525c8bb3209d01b17c322.patch

**Comment by crucialfelix** [comment]
beep beep.  I don't mind either way. but if its -V then that patch needs to be a PR so we can merge it

**Pull request merged**